src/test/scala/mesosphere/marathon/core/task/tracker/impl/TaskStateOpResolverTest.scala

```diff
@@ -23,15 +25,129 @@ class TaskStateOpResolverTest
   test("an update for a non-existing tasks is mapped to fail") {
     val f = new Fixture
     Given("a taskID without task")
-    val appId = PathId("/app")
-    val taskId = Task.Id.forApp(appId)
-    f.taskTracker.task(taskId) returns Future.successful(None)
+    f.taskTracker.task(f.notExistingTaskId) returns Future.successful(None)
 
     When("resolve is called")
-    val stateChange = f.stateOpResolver.resolve(TaskStateOp.ForceExpunge(taskId)).futureValue
+    val stateChange = f.stateOpResolver.resolve(TaskStateOp.ForceExpunge(f.notExistingTaskId)).futureValue
+
+    Then("getTaskAsync is called")
```

Comment:
consistent with the rest of the tests -- but the method is not called getTaskAsync anymore.